tests/saitests/py3/sai_qos_tests.py

```diff
@@ -406,7 +469,7 @@ def runTest(self):
                         continue
 
             # Read Counters
-            time.sleep(10)
+            time.sleep(3)
```

Comment:
I have reduced the time because the longer time allows LACP packets to get enqueued and cause the counters to go up.